tests/test_electrolyzer.py

```diff
@@ -187,11 +175,154 @@ def energy_system_options(self):
         )
         #  -----------------------------------------------------------------------------------------
 
+    def test_electrolyzer_constant_efficiency(self):
+        """
+        This test is to check the functioning the example with an offshore wind farm in combination
+        with an electrolyzer and hydrogen storage. The electrolyzer is modelled as the option
+        CONSTANT_EFFICIENCY.
+
+        Checks:
+        - The objective value with the revenue included
+        - Check the bounds on the electrolyzer
+        - Check the setpoint for the windfarm
+        - Check the max production profile of the windfarm
```

Comment:
I don't think it is necessary to check all of these values again, the main focus of this test is to check that the electrolyzer efficiency is constant and that is calculated correctly.

Reply:
Reduced test set to just contain the constant efficiency check

Reply:
Yes that is perfect, one small one to add, is a check on the electricity power conservation, this ensures that the answer is also always reasonable. If you add a TODO for this that is fine, as I have generalized part of these checks which are in a PR for this week.